KerbalAcademyBootCamp.cfg

```diff
@@ -119,13 +126,13 @@
 			define = BootCamp
 			disableOnStateChange = true
 			dissassociateVesselsOnContractCompletion = true
-			title = Recruit @/roleString Then place them in a vessel capable of completing this mission.
+			title = Recruit @/roleStringSingular, then place them in a vessel capable of completing this mission.
```

Comment:
does the comma stop the contract from loading into the game? Has this been checked? @karnwine729